pyaedt/edb_core/padstack.py

```diff
@@ -470,6 +470,51 @@ def get_pad_parameters(self, pin, layername, pad_type=0):
         rot = padparams[5].ToDouble()
         return geom_type, parameters, offset_x, offset_y, rot
 
+    @pyaedt_function_handler
+    def set_all_antipad_value(self, value):
+        """Set all anti-pads from all pads-tack definition to the given value.
+
+        Parameters
+        ----------
+        value float, str
+            anti-pad value
+
+        Returns
+        --------
+        bool
+            ``True`` when all succeed ``False`` if one of anti-pads value failed to be assigned.
+        """
+        if self.padstacks:
+            for padstack in list(self.padstacks.values()):
+                cloned_padstack_data = self._edb.Definition.PadstackDefData(padstack.edb_padstack.GetData())
+                layers_name = cloned_padstack_data.GetLayerNames()
+                all_succeed = True
+                for lay in layers_name:
```

Comment:
Maybe you can rename `lay` in `layer`.
```suggestion
                for layer in layers_name:
```

Reply:
I you decide to do so, you will have to update it in the loop below also. Your call 😃

Reply:
yes I did thx